server/db/models/cart.js

```diff
@@ -0,0 +1,19 @@
+const Sequelize = require('sequelize');
+const db = require('./../database.js');
+
+const { UUID, UUIDV4, INTEGER } = Sequelize;
+
+const Cart = db.define('cart', {
+    id: {
+        primaryKey: true,
+        type: UUID,
+        defaultValue: UUIDV4
+    },
+    productQuantity: {
+        type: INTEGER,
+        allowNull: false,
+        defaultValue: 0,
```

Comment:
Do we want to make default value 1?

Reply:
Why 1?

Reply:
I don't think people will add something to cart if they don't want any of it.

Reply:
Maybe I just remove the default value setting in the model? Seems it's not necessary. The product should be removed from the Cart table if the quantity is zero. Does this sound right?